app/controllers/task_controller.rb

```diff
@@ -1,33 +1,77 @@
 class TaskController < ApplicationController
```

Comment:
Per Rails naming conventions, you'd want to use a plural controller name TasksController